src/app/site/site.component.html

```diff
@@ -1,10 +1,10 @@
 <div>
-    <div class="fixed">
-        <app-header *ngIf="!isMobile"></app-header>
-        <app-mobile-header *ngIf="isMobile" [menuWidth]="curWidth - 40" [menuHeight]="curHeight"></app-mobile-header>
-    </div>
-    <div [style.min-height]="(curHeight - 210) + 'px'" style="margin-top: 70px">
-        <router-outlet></router-outlet>
-    </div>
-    <app-footer></app-footer>
-</div>
\ No newline at end of file
+  <div class="primary fixed">
+    <app-header *ngIf="!isMobile"></app-header>
+    <app-mobile-header *ngIf="isMobile"></app-mobile-header>
```

Comment:
add [menuWidth]="curWidth - 40" [menuHeight]="curHeight" after *ngIf; so this inputs are necessary and defined in .ts related file

Reply:
Same as above